build/validateNodeVersion.js

```diff
@@ -0,0 +1,112 @@
+const https = require('https');
+
+const SUPPORTED_LTS = "Carbon";
+const VERSION = process.versions.node;
+const NODE_JS_VERSION_LIST_URL="https://nodejs.org/dist/index.json"
+const MIN_MAJOR_SUPPORT = 8;
+const MIN_MINOR_SUPPORT = 9;
+const MIN_PATCH_SUPPORT = 4;
+
+let body = '';
+
+function fail() {
+    body = '';
+    console.log(false);
```

Comment:
Do we need to throw an error at some point to get the release script to break?

Reply:
console.log() is picked up by the bash script that invokes this. Do you think we also need to throw an error if the parent bash script is halting all execution after receiving a `false` signal from this?